### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by abotzki**
not sure whether here is a good place for the banner?

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -9,6 +9,10 @@ sidebar: false
 
 {% include index/welcome.html %}
 
+{% block announce %}
```
Comment (mihai-sysbio) [inline index.md:12]: `block` is not quite a Liquid term from what I can see.

When trying to view this in GitPod there is an error:
```
Liquid syntax error (line 5): Unknown tag 'block' in index.md
failed to load command: jekyll (/usr/local/bundle/bin/jekyll)
```

Any clue to what might be wrong?

Comment (abotzki) [reply index.md]: Sorry, did not test it before.

**Comment by abotzki** [comment]
changed the approach with an additional html file banner and define one extra custom class for red text
@mihai-sysbio thanks for your feedback!

File: _includes/index/beta-banner.html [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<div class="row justify-content-center">
+    <div class="col-12 text-center">
+        <div class="display-4 text-black fw-bold">
+            <p class="lead text-red mb-0">
+                This website is under active development and considered beta. Release: December 2024
```
Comment (mihai-sysbio) [inline _includes/index/beta-banner.html:5]: The above looks like this:
image

I would propose the following changes
```suggestion
<div class="bg-warning" style="position: relative; left: 50%; width: 100vw; transform: translate(-50%);">
    <div class="text-center">
        <div class="display-4 fw-bold p-4">
            <p class="lead text-red mb-0">
                This website is under active development and considered beta. <br> Planned release: December 2024
```

to obtain this look:
image

**Review by mihai-sysbio — APPROVED** [review]
Good idea with a banner component. It looks like it's not as easy to add it on other pages besides the homepage, but perhaps that's enough.

File: _includes/index/beta-banner.html (hunk 2 above)
Comment (abotzki) [reply _includes/index/beta-banner.html:5]: yes, that looks very nice. Thanks @mihai-sysbio

**Review by ninanorgren — APPROVED** [review]
Looks really nice!

**Pull request merged**